src/components/PrimaryNav.js

```diff
@@ -78,6 +83,8 @@ const PrimaryNav = forwardRef(
           justify="flex-end"
           flexGrow={1}
           hidden={!isVisible || undefined}
+          aria-hidden={isVisible}
```

Comment:
We likely need to add a check in here to see if Hamburger menu is visible, if it's not visible (ie. on Desktop) this needs to be set to `false` to ensure that folks on larger screen sizes, using AT, know the state of the nav.

Reply:
With this current declaration `aria-hidden` defaults to`false` on desktop. Do you still think an additional check needs to be added?
<img width="1055" alt="Screen Shot 2020-06-07 at 7 57 03 PM" src="https://user-images.githubusercontent.com/6998954/83983036-11f7d880-a8f9-11ea-9440-0178a47c49d6.png">

Reply:
@M0nica apologies, looking at the deploy preview, it appears that `aria-hidden` is set to `true` on desktop currently. We want to make sure this is `false` on desktop.

As soon as that medium breakpoint hits, and the navigation becomes hidden behind the hamburger menu, we want to make sure this switches to `true` until a user toggles the nav menu. Then it'll be toggled between `false` and `true` when the user hides / shows the nav.

Reply:
I realized I read that backwards 🤦🏾‍♀️. `aria-hidden` should now be false for NavMenu on desktop screens.

Reply:
@M0nica looks great! Thank you!